backend/authStrategies/microsoft.ts

```
@@ -25,44 +26,48 @@ interface msProfile {
+          fk: { microsoft: profile._json.id },
+          email: email,
+          name: {
+            familyName: profile._json.surname || (displayNameSplit.shift() as string), // first part of displayNameSplit
+            givenName: profile._json.givenName || (displayNameSplit.length > 1 ? displayNameSplit.join(' ') : ' ') // rest of displayNameSplit
+          }
+        }
+        if (!user) {
+          user = new User(newUser)
+        } else {
+          Object.assign(user.fk, newUser.fk)
+          delete newUser.fk
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)